canonical-web-and-design/react-components
# Initial build of ModularTable component

## Done

Added new `ModularTable` component that is a first basic implementation of the Vanilla modular table.
This is a MVP implementation of static table rendered based on `columns` and `data` objects.
It uses [React Table](https://react-table.tanstack.com/) internally for table functionality and data manipulation.

## QA

### Storybook

To see rendered examples of all react-components, run:

```shell
yarn start
```

### QA in your project

from `react-components` run:

```shell
yarn build
npm pack
```

Install the resulting tarball in your project with:

```shell
yarn add <path-to-tarball>
```

### QA steps

- Check ModularTable in Story book
- Make sure it renders correctly and is properly documented
